server_manager/electron/package.action.sh

```diff
@@ -16,9 +16,10 @@
 set -eu
 
 SCRIPT_DIR="$(dirname "$0")"
-PROJECT_DIR="${BUILD_DIR}/server_manager/electron/static"
+PROJECT_DIR=
```

Comment:
Why do you have this empty assignments?

Reply:
Just remove them from here.
The assignment in main will already set the global, because variables in bash are global by default.

This assignment prevents the detection of variables being used before being defined.